### DISCUSSION TITLE
Requests: Ability to modify fields + drag/drop order of fileds

### DISCUSSION THREAD

**Discussion by f3st3r113** [docusealco/docuseal#99] [Ideas]
Looking for the ability to modify fields, i.e. center text field, set font size, etc.

Looking for the ability to drag/drop filed order (in lieu of clicking up/down)

**Comment by AlexBTurchyn** [comment]
@f3st3r113 Converting this to discussions>ideas - the more upvotes it gets the sooner we will have this implemented :)
